### ISSUE TITLE
impossible to remove the icon

### ISSUE THREAD

**Issue by slicejunk** [arfedulov/semantic-ui-calendar-react#74]
**Describe the bug**

icon={false} does not remove the icon at all
icon={null} removes the icon, but adds an empty space before the input

**To Reproduce**

icon={false/null} on the imput

**Expected behavior**
no icon, no space

**Screenshots**
icon={null} produces:
![image](https://user-images.githubusercontent.com/5887100/51250966-67e07f00-1998-11e9-9133-f859caf7fbae.png)

**Dependencies versions**

* semantic-ui-react: 0.84.0
* semantic-ui-calendar-react: v0.13.0

**Comment by omarglz** [comment]
a quick fix can be overriding the css like so:
`.calendar.icon {
  display: none !important
}`

**Comment by arfedulov** [comment]
```jsx
  <DateInput icon={null} iconPosition="any str value except empty string and 'left'" />
```
This works for me.

**Comment by arfedulov** [comment]
fixed in #101
